core-bundle/assets/backend.js

```diff
@@ -0,0 +1,21 @@
+import { Application } from '@hotwired/stimulus'
+import { definitionForModuleAndIdentifier, identifierForContextKey } from '@hotwired/stimulus-webpack-helpers'
+
+import './scripts/autofocus'
```

Comment:
Just wondering if we shouldn't put these into some `backend` folder as well? Or where would the FE scripts go?

Reply:
I did think about that. We can always move scripts somewhere else and adjust the build scripts.